src/lib/viewers/image/__tests__/MultiImageViewer-test.js

```diff
@@ -232,7 +232,7 @@ describe('lib/viewers/image/MultiImageViewer', () => {
         });
     });
 
-    describe('setOriginalImageSize()', () => {
+    describe('setOriginalImageSizes()', () => {
```

Comment:
Should the child tests now invoke `multiImage.setOriginalImageSizes()`, instead?